packages/discount-code-generator/src/main.js

```diff
@@ -1,3 +1,62 @@
-export default function findSum (a, b) {
-  return a + b
+/* @flow */
+import TokenGenerator from 'tokgen'
+
+import type {
+  CodeData,
+  CodeDataArray,
+  CodeOptions,
+} from 'types/sdk'
```

Comment:
I think the `types/sdk` file is getting large, separate out your type declarations to another file...